internal/server/analytics/prometheus/client.go

```diff
@@ -44,7 +44,8 @@ func New(logger *zap.Logger, cfg *config.Config) (*client, error) {
 
 func (c *client) GetFlagEvaluationsCount(ctx context.Context, req *panalytics.FlagEvaluationsCountRequest) ([]string, []float32, error) {
 	query := fmt.Sprintf(
-		`sum(increase(flipt_evaluations_requests_total{namespace="%s", flag="%s"}[%dm])) or vector(0)`,
+		`sum(increase(flipt_evaluations_requests_total{environment="%s", namespace="%s", flag="%s"}[%dm])) or vector(0)`,
```

Comment:
I tried to run it and I see that `/metrics` endpoint returns this data

```
flipt_evaluations_requests_total{flipt_flag="demo",flipt_namespace="default",otel_scope_name="go.flipt.io/flipt/v2",otel_scope_version=""} 
```

There is no environment attribute. Should `flag` and `namespace` attributes be adjusted as well?

Reply:
good shout, yes I changed to `flipt_` as to not conflict with k8s metrics like `namespace` as we've gotten issues about this before. Will adjust. Not sure why `flipt_environment` is not showing though..

Reply:
It looks like https://github.com/flipt-io/flipt/blob/648e9b8b7b17fc79057c4f6413552ff145a3935d/internal/server/evaluation/evaluation.go#L86 doesn't include it